client/components/Item/MediaStreamSelector.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <v-select
+    v-model="trackIndex"
+    outlined
+    filled
+    flat
+    dense
+    single-line
+    hide-details
+    class="text-truncate"
+    :items="selectItems"
+    @input="$emit('input', $event)"
```

Comment:
This needs to return the `Index` property of the selected `MediaStream`.

Essentially what was done in the previous file:

```
if (this.tracks?.[newVal]?.Index) {
    this.$emit('input', this.tracks?.[newVal]?.Index);
} else {
    this.$emit('input', -1);
}
```

Reply:
This is already done in the `selectItems` computed property :)